Examples/AWSLambda.swift

```diff
@@ -0,0 +1,41 @@
+// begin lambda connection example 1
```

Comment:
I think it would be good to put this in a little swift package like our other examples, so we can show what dependencies/versions people need to depend on, and also so we can test that the code continues to compile.
there's a step in our release script that builds our existing examples via calling build-examples.sh; we should add that script to include this example too